tests/clone3-report-ns-id.c

```diff
@@ -0,0 +1,42 @@
+/*
+ * Check appending auxstr of clone3(2) when --namespace=switchTo is given
+ *
+ * Copyright (c) 2024 The strace developers.
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ */
+
+#include "tests.h"
+#include "xmalloc.h"
+
+#include <limits.h>
+#include <linux/sched.h>
+#include <stdio.h>
+#include <sys/syscall.h>
+#include <unistd.h>
+
+#include "scno.h"
+
+int
+main(void)
+{
+	struct clone_args arg = { .flags = CLONE_NEWUSER };
+	int pid = syscall(__NR_clone3, &arg, sizeof(arg));
+	if (pid < 0)
+		perror_msg_and_fail("clone3");
```

Comment:
When `clone3` syscall is not available, let's skip the test instead of failing it.

Reply:
fixed locally.